dandi/dandiapi.py

```diff
@@ -0,0 +1,348 @@
+from contextlib import contextmanager
+
+import requests
+
+from . import get_logger
+from .utils import ensure_datetime
+from .consts import MAX_CHUNK_SIZE
+
+lgr = get_logger()
+
+
+class HTTPError(requests.HTTPError):
```

Comment:
Why not just import HTTPError instead of making a do-nothing subclass?

Reply:
I wanted to aggregate non-standard exceptions our code might raise within this module.  I guess I could have just gone with `from requests import HTTPError` into this namespace indeed but then linters would not be happy (not used etc), so I somewhat like this explicit subclassing.  Any disadvantage?

Reply:
There's no real disadvantage other than a tiny bit of overhead.  I'd personally prefer to import HTTPError and add a `noqa` comment to tell flake8 to be quiet.